db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2025_08_27_205335) do
+ActiveRecord::Schema[7.1].define(version: 2025_11_26_005628) do
```

Comment:
You deleted the migration with this timestamp, didn't you?

Was that intentional?